pkg/pool-linear/contracts/LinearPool.sol

```diff
@@ -420,10 +426,11 @@ contract LinearPool is BasePool, IGeneralPool, LinearMath, IRateProvider {
         // triggered. This allows for a simple and safe way to exit the Pool.
         if (kind == ExitKind.EXACT_BPT_IN_FOR_TOKENS_OUT) {
             _ensurePaused();
-            // Note that this will cause for the user's BPT to be burned, which is not something that happens during
+            // Note that this will cause the user's BPT to be burned, which is not something that happens during
             // regular operation of this Pool, and may lead to accounting errors. Because of this, it is highly
             // advisable to not continue using a Pool on which the pause has been turned on and BPT burned once the
             // pause window expires.
+            // advisable to stop using a Pool after it is paused and the pause window expires.
```

Comment:
Repeated sentence?

Reply:
Yes - fixed in next commit. Too much copy/pasting to split this into so many PRs